internal/config/constants.go

```diff
@@ -50,7 +50,7 @@ const (
 	ProtobufTSGatewayVersion = "v1.1.2"
 	// ProtobufTSVersion is the version of protoc.
 	// renovate: datasource=npm depName=ts-proto
-	ProtobufTSVersion = "v1.115.5"
+	ProtobufTSVersion = "1.125.0"
```

Comment:
looks like `v` got dropped here. I don't know if that's intentional?

Reply:
yes, that's intention, `npm install` doesn't really need a `v` prefix to version, so renovate fixed that